docs/usage.md

```diff
@@ -0,0 +1,86 @@
+# Usage
+
+Use the import tool from the command line, like so:
+```
+java -jar <path-to-jar> <mode> <list of mcf/tmcf/csv files>
+```
+
+Hint: it can be useful to create an alias for the jar file, such as:
+```
+alias dc-import='java -jar <path-to-jar>'
+```
+
+This is the form that will be used in the rest of the documentation.
+
+Hint: to access a concise explanation of usage modes and flags, run
+`dc-import --help`
+
+## Usage Modes
+
+The import tool can either lint StatVars and other schema files, or
+it can generate instance MCFs from a TMCF and CSV.
+
+### Lint Mode (`lint`)
+To run the tool in lint mode, use:
+```
+dc-import lint <list of mcf files>
+```
+
+### Instance MCF Generation Mode (`genmcf`)
+To run the tool in genmcf mode, use:
+```
+dc-import genmcf <list of csv/tmcf files>
+```
+
+Note that in this mode, schema file(s) are optional to pass in.
+
+## Flags
+Flags available for the tool are listed below.
+
+You can also run `dc-import --help` to see a list of flags in your terminal.
+### `-e`, `--existence-checks`
+Check DCID references to schema nodes against the KG and locally. If set, then
```

Comment:
"Checks DCID references..."  (elsewhere too)... i.e., these might read better by using style guide suggestions for functions:

Function comments should be written with an implied subject of *This function* and should start with the verb phrase; for example, "Opens the file", rather than "Open the file".

Reply:
done. should I also update Main.java to modify the CLI output for --help?

Reply:
Good idea.  Lets do that in a separate PR though.

Reply:
done in #132